core/controllers/topics_and_skills_dashboard.py

```diff
@@ -58,8 +58,8 @@ def get(self):
         skill_summary_dicts = [summary.to_dict() for summary in skill_summaries]
 
         self.values.update({
-            'skills_list': skill_summary_dicts,
-            'topics_list': topic_summary_dicts
+            'skill_summaries': skill_summary_dicts,
```

Comment:
Sorry, I think I gave you the wrong advice here. I just realized that probably skill_summary_dicts and topic_summary_dicts would be better keynames, because the stuff being passed is neither a domain object in the frontend or the backend.

Reply:
Done.